summerofcode/2023.md

```diff
@@ -26,12 +26,42 @@ If you are a project maintainer and consider mentoring during the GSoC 2023 cycl
 
 ## Ideas
 
+- [Cloud Native Buildpacks](#cloud-native-build-packs)
```

Comment:
let's remove the ToC stuff for now. We can add it back at the end of the process.